### PR TITLE
fix: Remove + from jobs.image.tag if found

### PR DESCRIPTION
**Pull request by slushysnowman**
closes: #500

### CODE REVIEW

**Comment by slushysnowman** [comment]
Tested it with EKS v1.21+ - no issues, job images are able to be pulled successfully

File: charts/capsule/templates/_helpers.tpl [hunk 1]
```diff
@@ -98,7 +98,7 @@ Create the jobs fully-qualified Docker image to use
 {{- if .Values.jobs.image.tag }}
 {{- printf "%s:%s" .Values.jobs.image.repository .Values.jobs.image.tag -}}
 {{- else }}
-{{- $kubeversion := print "v" .Capabilities.KubeVersion.Major "." .Capabilities.KubeVersion.Minor -}}
+{{- $kubeversion := print "v" .Capabilities.KubeVersion.Major "." (.Capabilities.KubeVersion.Minor | replace "+" "") -}}
```
[the remark this answers is not in the document]
Comment (slushysnowman) [reply charts/capsule/templates/_helpers.tpl:101]: Yep, I'll sort it out

[the remark this answers is not in the document]
Comment (slushysnowman) [reply charts/capsule/templates/_helpers.tpl]: I've pushed a change for this, but honestly I'm not sure if it's better - but maybe I'm just doing it in an illogical way - fedeback welcome

[the remark this answers is not in the document]
Comment (ptx96) [reply charts/capsule/templates/_helpers.tpl]: @slushysnowman first of all, thank you for your contribution.

TBH with you, I can't find it very elegant to break the `$kubeversion` string parsing between helper.tpl and the template file ;
could you move also the `.Capabilities.KubeVersion.Major` inside the .tpl and just return the complete string? (maybe as "capsule.jobstagkubeversion")

[the remark this answers is not in the document]
Comment (slushysnowman) [reply charts/capsule/templates/_helpers.tpl]: @ptx96 Yep agreed, not very elegant! I've pushed a change, let me know what you think

**Review by ptx96 — APPROVED** [review]
Tested on labs, ok!  LGTM!

Waiting also for @prometherion

**Review by prometherion — APPROVED** [review]

**Comment by prometherion** [comment]
Thanks @slushysnowman, really appreciated it! 🚀

**Pull request merged**